recipe/recipe.yaml

```diff
@@ -0,0 +1,72 @@
+schema_version: 1
+
+context:
+  name: boost-histogram
+  version: 1.5.1
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/b/boost_histogram/boost_histogram-${{ version }}.tar.gz
+  sha256: 6e0f3dbdf7ab60ca1c61d5caa4f3f9e6097461feb65c6dbb01302fd88b1be524
+
+build:
+  number: 1
+  script:
+    - if: not (linux and ppc64le)
+      then: export CMAKE_BUILD_PARALLEL_LEVEL=${CPU_COUNT}
+    - ${{ PYTHON }} -m pip install . -v
+
+requirements:
+  build:
+    - if: build_platform != target_platform
+      then: python
```

Comment:
FYI, you can merge these into a single element:

```yaml
    - if: build_platform != target_platform
      then:
        - python
        - cross-python_${{ target_platform }}
```

Unfortunately the conda-recipe-manager tool doesn't do that yet.